### DISCUSSION TITLE
Similar plugin for Loupedeck?

### DISCUSSION THREAD

**Discussion by Limeqvist** [JaouherK/streamDeck-weatherPlugin#58] [General]
Hi Jaouher!

I was wondering if you would be up for doing a similar plugin for us at Loupedeck, I could provide you with e.g. a Loupedeck Live device :)

**Comment by JaouherK** [comment]
Hi @Limeqvist ,
Thank you for reaching out 👍 
Looks like a very interesting idea. I am excited to create similar plugins for Loupedeck. If you want, we can continue this discussion through your profile email.
Best regards
